Insurgency/insserver

```diff
@@ -3,46 +3,87 @@
 # Server Management Script
 # Author: Daniel Gibbs
 # Website: http://gameservermanagers.com
-version="291015"
+version="031115"
 
 #### Variables ####
 
-# Notification Email
-# (on|off)
-emailnotification="off"
-email="email@example.com"
+# The name of this script file, used to show the LGSM link properly
+selfname=$(basename $(readlink -f "${BASH_SOURCE[0]}"))
```

Comment:
Because this will find the real file name of the script, even if a user creates hardlinks instead of symlinks for the multiple instances.